Resources/Prototypes/Guidebook/chef.yml

```diff
@@ -10,10 +10,11 @@
   name: guide-entry-cookbook
   text: "/Server Info/Guidebook/Chef/cookbook.xml"
   children:
+  - CookBurger
+  - CookSauce
   - CookSandwich
   - CookSnack
```

Comment:
Place on top of the list, it's our catch-all and I would like to lower the chance of it being dismissed. Don't have a better name for it yet, though.

Reply:
I did mean snacks.